docs/en_US/install/3ds.md

```diff
@@ -83,19 +83,25 @@ Depending on how you installed Nimbus, launch it either through the Homebrew Lau
 
 Your selection persists between reboots.
 
-## Signing into your PNID
+## Signing in into Pretendo Network
 
-The 3DS does not rely on NNIDs for the vast majority of it's game servers. Because of this, using a PNID is also not required for most games<sup><a>[[1]](#footnote-1)</a></sup>.
-
-Setting up a PNID on the 3DS is the same as setting up a NNID. You may either create the PNID on your console, or register from an account [on our website](/account/register) and link it to your console once you're ready.
-
-It is recommended to register the PNID on your device at this time, as registering on the website does not currently allow you to change your user data.
+On your system, go to **System Settings** and click on **Nintendo Network ID Settings**. From there, follow the instructions on screen to create a new Pretendo Network ID or link an existing one created [on our website](/account/register).
```

Comment:
One of your stated goals was to make these sections less wordy, but by adding this section you seem to have gone against that? This originally just informed users the setup process was exactly the same as a NNID, which told them everything they needed to know. Restating that here seems to just be repetitive for no valuable reason?

Also since the 3DS does not require a network account for basically all titles, the fact that you do not need a PNID was intentionally mentioned *first*, so that users could skip these steps if need be

All this has done is reorder the steps to make it somewhat less intuitive imo

Reply:
While it is mentioned that it is the same steps as setting up NNID, passage lacks information on **where** to create or link a NNID on the console. As creating an NNID is a once-and-forget thing, I believe that adding the location on where to do this helps a lay person.

> Also since the 3DS does not require a network account for basically all titles, the fact that you do not need a PNID was intentionally mentioned first, so that users could skip these steps if need be

I would argue that the information provided is not clear enough as the structure of the text conveys that it is the next step after using Nimbus. Putting this step in a separate "extras" section or clearly marking the step as optional in the heading might solve this.